packages/nextra-theme-docs/css/styles.css

```diff
@@ -34,6 +34,19 @@ summary {
   @apply nx-rounded;
 }
 
+.nextra-content {
+  @apply nx-text-slate-700 dark:nx-text-slate-200;
+
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6 {
+    @apply nx-text-slate-900 dark:nx-text-slate-100;
```

Comment:
you can add them directly in `mdx-components.tsx`
here
https://github.com/shuding/nextra/blob/0cb1f35999b75bda5406e6212e4a40a5e9111cc0/packages/nextra-theme-docs/src/mdx-components.tsx#L234
and here
https://github.com/shuding/nextra/blob/0cb1f35999b75bda5406e6212e4a40a5e9111cc0/packages/nextra-theme-docs/src/mdx-components.tsx#L110

Reply:
moved to `mdx-components.tsx`